### DISCUSSION TITLE
IEEE 802.11 Frame Aggregation - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#621] [Showcases]
Discuss the "IEEE 802.11 Frame Aggregation" showcase here. Ideas and recommendations are welcome.

**Comment by aditi89** [comment]
Is their any possibility one can do packet aggregation at the network layer through this?

**Comment by levy** [comment]
Aggregation in general is not difficult [see here](https://github.com/inet-framework/inet/blob/topic/protocol/src/inet/protocol/Aggregation.cc) and [here](https://github.com/inet-framework/inet/blob/topic/protocol/src/inet/protocol/Deaggregation.cc)

**Comment by aditi89** [comment]
> 
> 
> Aggregation in general is not difficult [see here](https://github.com/inet-framework/inet/blob/topic/protocol/src/inet/protocol/Aggregation.cc) and [here](https://github.com/inet-framework/inet/blob/topic/protocol/src/inet/protocol/Deaggregation.cc)

The above links are not working.can you please activate these links again.

**Comment by levy** [comment]
File are [moved](https://github.com/inet-framework/inet/tree/topic/protocol/src/inet/protocol/aggregation)

**Comment by aditi89** [comment]
> 
> 
> File are [moved](https://github.com/inet-framework/inet/tree/topic/protocol/src/inet/protocol/aggregation)

source files of aggregation and de aggregation are missing in this link.
